tests/integration/io/parquet/test_read_row_groups.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+import pandas as pd
+import pytest
+
+import daft
+
+FILES = [
+    "tests/assets/parquet-data/mvp.parquet",
```

Comment:
We should also add some of the s3 file sources here too. I believe we all these files in s3 as well

Reply:
I just removed this test and added a fixture to `test_reads_public_data.py`